### PR TITLE
update the doc about hosting powershell with a sample that uses beta.1 packages

### PR DESCRIPTION
**Pull request by daxian-dbw**

### CODE REVIEW

**Comment by daxian-dbw** [comment]
@adityapatwardhan could you please take a look? This PR adds a sample to show how to hos powershell with `beta.1` packages.

File: docs/host-powershell/README.md [hunk 1]
```diff
@@ -155,4 +164,4 @@ The issue [#3417](https://github.com/PowerShell/PowerShell/issues/3417) was open
 
 [CorePsAssemblyLoadContext.cs]: https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.CoreCLR.AssemblyLoadContext/CoreCLR/CorePsAssemblyLoadContext.cs
 [Resolving]: https://github.com/dotnet/corefx/blob/ec2a6190efa743ab600317f44d757433e44e859b/src/System.Runtime.Loader/ref/System.Runtime.Loader.cs#L35
-[bootstrapping]: https://github.com/PowerShell/PowerShell/blob/master/src/powershell/Program.cs#L27
\ No newline at end of file
+[bootstrapping]: https://github.com/PowerShell/PowerShell/blob/master/src/powershell/Program.cs#L27
```
Comment (adityapatwardhan) [inline docs/host-powershell/README.md]: No change here?

Comment (daxian-dbw) [inline docs/host-powershell/README.md]: I hate VSCode sometimes 😆

**Comment by daxian-dbw** [comment]
@adityapatwardhan it's fixed. It turns out that was because a new line was added to the end of that line.

**Review by adityapatwardhan — APPROVED** [review]
LGTM

**Pull request merged**